File: C#/Example Statement/Example If, Else If, dan Else/Nested If dan Else.cs
using System;

namespace Main
{
    class main
    {
        static void Main(string[] args)
        {
            string data1, data2;

            Console.WriteLine("Login :\n");
            Console.Write("username : ");
            data1 = Console.ReadLine();
            Console.Write("password : ");
            data2 = Console.ReadLine();

            if (data1 == "user1")
            {
                if (data2 == "psw123")
                {
                    Console.WriteLine("\nyou succeeded");
                }
                else
                {
                    Console.WriteLine("\nwrong password");
                }
            }
            else
            {
                Console.WriteLine("\nwrong username");
            }

        }
    }
}

File: C++/Example Statement/Example If, Else If, dan Else/Nested If dan Else.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
	string data1, data2;
	cout << "Login \n" << endl;
	cout << "username : ";
	cin >> data1;
	cout << "password : ";
	cin >> data2;

	if (data1 == "user1") {
		if (data2 == "psw123") {
			cout << "\nyou succeeded" << endl;
		}
		else {
			cout << "\nwrong password" << endl;
		}
	}
	else {
		cout << "\nwrong username" << endl;
	}

	return 0;
}
